Know-Your-Nation-Speedy/Know-Your-Nation-Speedy/Models/Articles.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Threading.Tasks;
+namespace Know_Your_Nation_Speedy.Models
+{
+    public class Articles
```

Comment:
You should give classes singular names rather than plural, this can be done to all the classes defined below

Reply:
was told to make it plural in order to keep variables made from the classes in a better naming convention, eg Users User